app/views/govuk_publishing_components/components/_warning_text.html.erb

```diff
@@ -15,8 +18,15 @@
   <% unless text_icon.empty? %>
     <%= tag.span text_icon, class: "govuk-warning-text__icon", "aria-hidden": "true" %>
   <% end %>
-  <%= tag.strong class: text_classes do %>
-    <%= tag.span text_assistive, class: "govuk-warning-text__assistive" %>
-    <%= text %>
+  <% if heading_level > 0 %>
```

Comment:
Would this need refining? Would it render `<h123>` with `heading_level: 123`?

Reply:
The shared helper code already handles this - if you pass anything invalid it defaults to a H2 - https://github.com/alphagov/govuk_publishing_components/blob/master/lib/govuk_publishing_components/presenters/shared_helper.rb#L22